android/app/proguard-rules.pro

```diff
@@ -18,7 +18,7 @@
 
 # Disabling obfuscation is useful if you collect stack traces from production crashes
 # (unless you are using a system that supports de-obfuscate the stack traces).
--dontobfuscate
+# -dontobfuscate
```

Comment:
I think we don't want to obfuscate release builds (see my comment above). Maybe it's fine to obfuscate when the crashlytics guide linked above is followed